test/wdio/package.json

```diff
@@ -3,7 +3,9 @@
   "type": "module",
   "version": "0.0.0",
   "scripts": {
-    "build": "node ../../bin/stencil build --debug --es5",
+    "build": "run-s build.main build.global-script",
```

Comment:
I thought about it, I don't think that the two builds will conflict or anything, but I thought that maybe the memory usage might be a little high on the gh actions runners so I went with sequential

just pushed a change to use `run-p` to see what it looks like

Reply:
you can see an example of the output here: https://github.com/ionic-team/stencil/actions/runs/8363897946/job/22898026360?pr=5542

I think I might want to stick with `run-s` here, not because of a performance issue but just I think the interleaved build output is somewhat confusing to look at